service/inject_definition.go

```diff
@@ -10,6 +11,10 @@ import (
 // injectDefinition applies service definition to Service type.
 func (s *Service) injectDefinition(def *importer.ServiceDefinition) {
 	s.Name = def.Name
+	s.Alias = def.Alias
+	if s.Alias == "" {
+		s.Alias = s.generateAlias()
```

Comment:
why are we forcing to have an alias? it might be better to only use ids when an alias is not provided by user. what's the purpose of creating an alias internally which is never used by the user?

Reply:
We are relaying on alias for docker volume, but yes we could do a better system that an generated alias.
We could create a alias getter that return it if if it exist or return the service id otherwise.
@antho1404 what do you think?

Reply:
we could also have a test on the docker volume part to take either the alias or the id but i'm pretty sure we might need this in other places so generating an alias is really convenient. Also I don't see any problem of having default properties, we already have something like that for the configuration

Reply:
Ok for me

Reply:
The problem with this is we're generating the alias from the _Name_ field of _mesg.yml_. In other means we're forcing users to define a unique _Name_. If we're going to create an alias anyway depending on the _Name_ why we have both of them, it's a bid odd. If this is an acceptable behavior, I'm ok with this.

Reply:
The alias (if omitted) is now generated from the service hash